_docs/configure/index.md

```diff
@@ -1,5 +1,48 @@
 ---
-title: Configure 
+title: Configuring Forseti
```

Comment:
nit: Forseti -- we're already at /configure/index.html

Reply:
Updated to Forseti.